### ISSUE TITLE
[Bug/balance] infinite diamond production using the mechnical squeezer

### ISSUE THREAD

**Issue by SuperJelle** [AllTheMods/ATM-3#501]
## Expected Behavior
The (mechanical) squeezer from integrated dynamics should produce charcoal dust from charcoal (like all other machines with similar functions) instead of pulverized coal.

## Current Behavior
The squeezer produces pulverized coal from charcoal. This allows for an infinite (free) diamond production.

## Steps to Reproduce

Assumptions for free infinite diamond production: 
1. Some sort of auto-harvesting setup of wood.
2. Renewable energy such as the wind generator.

Method:
1. Burn wood to get charcoal
2. Squeeze charcoal to get pulverized coal
3. Use the manufactory to convert coal into graphite dust
4. Smelt graphite dust into graphite bar
5. Use graphite bars in the pressurizer to get diamonds (64 bars per diamond)

**Comment by ouroborus** [comment]
I agree that you should get charcoal dust rather than coal dust from charcoal. However, what you describe isn't exactly free. You need the wood and the energy/time to run the various machines. Even without this particular charcoal to coal dust path, you could still do this via charcoal to torches, torches to coal, coal to coal dust.

**Comment by SuperJelle** [comment]
Im perfectly aware of the assumptions I already stated in my post - there is no 'time' requirement other than setup though as everything can be automated. There are indeed other ways (including just a sag mill and a redstone furnace) to produce infinite automated coal/xp/diamonds, I figured I'd just start with one and depending on the response add more afterwards.
Overall I think the value of rarities such as diamonds should be somewhat preserved though it may be difficult with a modpack this big.
